### ISSUE TITLE
Bad Chunk Error After Creating Pipeline on Beta 4

### ISSUE THREAD

**Issue by dgeorg42** [pachyderm/pachyderm#6640]
On a clean Beta 4 installation, I created one repo and one pipeline.  The pipeline pod immediately failed to start and went into a "crash backoff loop" of restarts.  User log shows a bad chunk error:
```
error getting pipelineInfo: could not retrieve pipeline spec file from PFS for pipeline 'benchmarking': bad chunk. HAVE: 30653537353163303236653534336232653861623265623036303939646161316431653564663437373738663737383766616162343563646631326665336138 WANT: 39356566306662386266653864316165366231623630303338326235643332633064383965343036366466333233646232636334663562316562646530656266
```

I tried deleting and recreating the pipeline without success.  This is the same pipeline that I have been using for my performance testing all along, so it should just work.

[storage.log](https://github.com/pachyderm/pachyderm/files/6897302/storage.log)
[user.log](https://github.com/pachyderm/pachyderm/files/6897303/user.log)

[AB#12](https://dev.azure.com/PachydermCompany/92b98a32-88e5-46cb-a28f-5d064712e03b/_workitems/edit/12)

**Comment by dgeorg42** [comment]
Here are the `pachd` logs as well:

[pachd.log](https://github.com/pachyderm/pachyderm/files/6897359/pachd.log)

**Comment by dgeorg42** [comment]
This morning I tried deploying a completely different pipeline, and got the same result.

**Comment by dgeorg42** [comment]
The pipeline I'm referring to in my previous comment was one of the customer pipelines I've been testing with called "cron_daily". After it failed in the same way as the pipeline I initially tried creating yesterday. I decided to try to uninstall and reinstall Pachyderm to see if that had any effect on the issue.  When I did, however, I forgot to delete the `cron_daily` pipeline first. So, after reinstall, that pipeline tried to start and failed in the same way again.  However, there was also a new problem that surfaced, specifically `pachd` started failing and restarting as well, with segmentation faults. I have opened #6641 to track that issue.

**Comment by dgeorg42** [comment]
This is reproducible on Hub as well:
```
error getting pipelineInfo: could not retrieve pipeline spec file from PFS for pipeline 'benchmarking': bad chunk. HAVE: 30653537353163303236653534336232653861623265623036303939646161316431653564663437373738663737383766616162343563646631326665336138 WANT: 64313038643461323032313737343739366137643536343864646264326637366639346234353865656365306433366162353339613263343636326666316438
```

**Comment by brycemcanally** [comment]
This should be addressed by: #6651. Feel free to reopen if it is not.

**Comment by dgeorg42** [comment]
Confirmed this is working on Beta 5.
